### PR TITLE
Microverse survey form project 

### PR DESCRIPTION
**Pull request by bernyhas**
This is the pull request for survey form microverse project any suggestion and changes are welcome

### CODE REVIEW

**Comment by bernyhas** [comment]
Hello @bernyhas,

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there! make sure you improve your html  semantic tag

**Comment by bernyhas** [comment]
Great job @bernyhas ,

Your project is complete! 
Congratulations! Hans

Cheers and Happy coding 

Feel free to leave any questions or comments !

File: index.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+              <label for="name">Name</label>
+              <input type="text" id="name" placeholder="Enter your name" />
+            </div>
+            <div class="form-group">
+              <label for="sex">Sex</label>
+              <select name="sex" id="sex">
+                <option value="male" selected>Male</option>
+                <option value="female">Female</option>
+                <option value="other">Other</option>
+              </select>
```
Comment (Temmarie) [inline index.html:34]: - [ ] According to the project's requirements, you have to:
- Use the GitHub inline comments feature at least once to leave review comments. For this project, kindly leave the comment under a line or block of code that fits the context the comment is about. This not only makes the comment complete but understandable as well. I will try to invite you to check this [lesson on how to add inline comments](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request).
- You have to write at least 2 comments in the review.

- [ ] You've made a polite request [here](https://github.com/bernyhas/Form-Survey/pull/1#issuecomment-875474864), but your suggestion lacks a `why`. Include an explanation of why you think the student should improve the semantic HTML tags. 

Please always include an explanation, a "why", for each requested change. For example:
- This is a well-explained request: 
  Kindly remove the `header` tag from being a child of the `main` tag, this helps to improve the semantic correctness of the HTML structure and making the page more accessible. 
- This request lacks a "why": 
Make sure you improve your Html semantic tag.

- [ ] A nice tip to consolidate your reviews under a single summary is to click the "Start a review" button when adding the first review comment. This helps to house all your inline comments under one single summary like this current review. :wink:
This is an example inline comment. :slightly_smiling_face:
You can see how it's under the main summary above and point to a specific part of the code where you want to suggest changes or just comment on it generally. :wink:
To do this, make sure you go to the "File changes" tab and use the "Start a review" button for your first review comment.

**Review by Temmarie — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED! :recycle:

### Hi @bernyhas      ! :wave: :wave:

#### You have done really well in implementing the requirements.
- You politely requested the changes ✔️ 
- You added a summary message ✔️ 

 However, there are changes required for you to make in order for this project to be approved. 
- - -

### **Required Changes** ♻️

_See the inline comments below for required changes_


### Optional Changes 👌
Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better.
- N/A

_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

**_"Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."_**
- - - 
**Note that you still have 2 more reviews as written in the [Code Reviews Limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561), however, if you feel this review is unfair, you can request a second opinion using [this](https://airtable.com/shrQAqnBwek5a0O0s) form.**

__Kindly make the necessary changes and submit another code review request.__

**Review by bernyhas — COMMENTED** [review]
Hi @bernyhas 

your html code has good so far request you to improve the code using html semantic tag!

**Review by bernyhas — COMMENTED** [review]
hi @bernyhas ,
your  css code is great please just request you to improve the base code.
if any question let me know in the comment below
Cheers Happy coding!

File: index.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+            <div class="form-control">
+              <label>What your Programming level skill:</label>
+              <input type="radio" name="membership" value="simple" />
+              Basic
+              <input type="radio" name="membership" value="standard" checked />
+              Intermediate
+              <input type="radio" name="membership" value="super" />
+              Advance
+            </div>
```
Comment (Davidosky007) [inline index.html:62]: * Like the previous code reviewer pointed you still have not used the Github inline comment feature
* This is an example inline comment. 🙂

You can see how it's under the main summary above and point to a specific part of the code where you want to suggest changes or just comment on it generally. 😉

To do this, make sure you go to the "File changes" tab and use the "Start a review" button for your first review comment.

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @bernyhas, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
# Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 2 above)
Comment (bernyhas) [inline index.html:62]: Hi @bernyhas 

Your Project really great so far and request you to improve  this code.

Happy coding!

File: style.css [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+}
+
+a {
+  text-decoration: none;
+}
+
+ul {
+  list-style-type: none;
+}
+
```
Comment (bernyhas) [inline style.css:21]: Hello @bernyhas ,

You almost done and looking great your project  in fact   can you improve your base css.

Cheers!

**Review by bernyhas — COMMENTED** [review]
Hi @bernyhas ,

Hopefully you understand and improve your code.

Thanks

**Comment by Davidosky007** [comment]
Hi @bernyhas, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

* You added a summary and an inline comment that is polite and explains why the change is requested. As in total you left more than 2 comments this project is approved :)
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**